modules/launcher/src/main/java/org/mule/module/launcher/artifact/AbstractArtifactClassLoader.java

```diff
@@ -24,12 +27,18 @@
  */
 public abstract class AbstractArtifactClassLoader extends FineGrainedControlClassLoader implements ArtifactClassLoader
 {
+    private static final String DEFAULT_LEAK_CLEANER_CLASS = "DefaultLeakCleaner.class";
```

Comment:
Can you use a full qualified class name instead? (ie: org.mule.module.launcher.artifact.DefaultLeakCleaner)